release/one_click_linux_gui/create_installer_linux.sh

```diff
@@ -17,7 +17,7 @@ python setup.py sdist bdist_wheel
 # Setting up the local package
 cd release/one_click_linux_gui
 # Make sure you include the required extra packages and always use the stable or very-stable options!
-pip install "../../dist/alphapept-0.3.29-py3-none-any.whl[stable]"
+pip install "../../dist/alphapept-0.3.29-py3-none-any.whl[gui-stable]"
```

Comment:
release/one_click_linux_gui/control is already at version 0.3.30. Check your dependabot setup for missing files...